src/Microsoft.DotNet.Build.Tasks/PackageFiles/test-runtime/project.json

```diff
@@ -1,33 +0,0 @@
-{
-  "dependencies": {
```

Comment:
I would like to delete it as well but I'm not sure if we are using it anywhere. @ellismg @mellinoe @StephenBonikowsky do any of you guys know if we are using this from build tools package any longer?